### ISSUE TITLE
(PRE) Reviewer 2 comments

### ISSUE THREAD

**Issue by ncclementi** [barbagroup/pygbe_lspr_paper#11]
In manuscript EC12092, Clementi et al. describe a boundary element method at the electrostatic limit to simplify the numerical calculation/analysis for bio-sensing using nanoplasmonics. Due to the conductive electrons of the metallic nano-particles (NP), the electric field/density of states will be significantly magnified at the hot spots, which in turn make the effective mode volume much smaller than the relatively more traditional optical cavities. As a result the light matter interaction is strongly enhanced when the target/matter is put around the NP, and this is the physical basis of most applications of nanoplamonics and the biosensing studied in the current manuscript. As a specific application of nanoplasmonic, the latest development in the field should be well referenced such as single molecule imaging Ref.[Nature 498, 82 (2013)] and the modal theory developed in the past few years Refs.[Phys. Rev. Lett. 110 237401 (2013); New J Phys. 16, 113048 (2014)]. With their own software (PyGBe) they could reduce the computational complexity as O(NlogN). Over all the manuscript is well presented with respect to its technical
details. 

- [x] However, the manuscript is made redundant at several places such as Subsection F in Section II and Subsection C in Section III; most of them could be moved the the documented files then make a reference to them at the reference list in order to make the manuscript more compact and readable. 

And the following are a number of issues/comments which need to be addressed:

- [x] 1. At the end of the manuscript, the authors claimed that for a distance (between the NP and the analytes ) of d = 0.5 nm the redshift of the peak 0.75 nm; but it seems at this regime the nonlocal effect could kick in Ref.[Nature Commun. 5, 3809 (2014)].

- [x] 2. The superindex in the definition of V and K in Eqs. (8-9) should be corrected.

- [x] 3. The bold fonts for pi between eq.(22) and eq.(24) should be changed into plain.

- [x] 4. Should the “location” in ”To study the effect of location of the analytes, ... Figure 13” (on page 9) be orientation?

- [x] 5. Figures (1) and (2) are basically the same which make the manuscript redundant. The same happens to Figs (3) and (7), and Figs (9) and (13)

- [x] 6. Is it possible to give more physical description of the bovine serum albumin proteins such as the distribution of the charge as implied in Eq.(12)? This I think may be useful to understand the effect of orientation of the molecule.

- [x] 7. On page 11 the authors wrote ”Figure 10 shows a red shift of the plasmon resonance frequency peak in presence of the BSA proteins. This result agrees with experimental observations [42,43] ...”. In fact the red shift is easy to understand from a two-mode model and assume the mode provided by the BSA proteins is a lower energy mode with respect to the pamonic mode. So it is hard to say the experiment support the numerical calculation here. Maybe it is better to add in new calculation with well established software/technique to support the numerics here; this is just a very personal suggestion, and I would leave it to the author to decide.

To summarize, the current manuscript is treating some interesting problem, but there are some questions need to be addressed. So I would not recommend any form of publication until the questions/comments are fully addressed.

**Comment by cdcooper84** [comment]
### Comment -1
> As a specific application of nanoplasmonic, the latest development in the field should be well referenced such as single molecule imaging Ref.[Nature 498, 82 (2013)] and the modal theory developed in the past few years Refs.[Phys. Rev. Lett. 110 237401 (2013); New J Phys. 16, 113048 (2014)].

#### Ideas
- We need to read these papers and references there. Looking at the title, they may be related with our work. 

#### Reply
The reviewer points out interesting references. The first one is related to our work regarding efforts towards single-molecule detection. We included this reference in the discussion section, where the issue of experimental evidence of low analyte concentration is discussed.

We believe the other two references are out of the scope to our work. These focus on the modal theory of quantum effects in nanoplasmonics, without referencing biosensing applications. In our work, we focus on nanoplasmonics for biosensing, and, moreover, we use a classical approach.

#### Modifications
We added reference to Nature 498, 82 (2013) in commit 58228689

**Comment by cdcooper84** [comment]
### Comment 0
>  However, the manuscript is made redundant at several places such as Subsection F in Section II and Subsection C in Section III; most of them could be moved the the documented files then make a reference to them at the reference list in order to make the manuscript more compact and readable.

#### Ideas
- These sections are related to (1) a summary of the modifications, and (2) reproducibility. I think the reviewer is just not used to see that and we should argue that it is important to include them. 
#### Reply
- §II.F. lists the code modifications and enhancements needed to produce the results in this paper, and we feel is necessary to include in the Methods portion of the paper. §III.C. is our standard statement on reproducibility of the results, which our group has made a pledge to always include. The sum of these two sub-sections is about one column of text, which hardly impacts the overall length. 

#### Modifications
No modifications needed

**Comment by cdcooper84** [comment]
### Comment 1
> 1. At the end of the manuscript, the authors claimed that for a distance (between the NP and the analytes ) of d = 0.5 nm the redshift of the peak 0.75 nm; but it seems at this regime the nonlocal effect could kick in Ref.[Nature Commun. 5, 3809 (2014)].

#### Ideas
- We need to read that paper. It looks at the interaction between two metallic nanoparticles, and we should find what they mean by nonlocal.
- Protein doesn't generate plasmons, hence there shouldn't be any coupling effect (not sure if that is what he/she means by nonlinear effect)
- In electrostatics, at 0.5nm the behavior is almost 'bulk-like'. In this case, the incoming field generates a 'dipole-like' effect on the nanoparticle which should decay faster. Can we argue our way with something like this?

After reading the paper https://www.nature.com/articles/ncomms4809 and its supplementary material
https://media.nature.com/original/nature-assets/ncomms/2014/140502/ncomms4809/extref/ncomms4809-s1.pdf

We found in the "Validity domain" section this statement "We point out that our diffusive model is valid for structural dimensions exceeding the mean-free path that in pure single crystals can be of the order of 100 nm for Ag and Au, down to ~3 nm for Na "

which we understand it is saying that their model for Ag and Au is valid if the structural dimensions are bigger than 100 nm. In our case the Ag NP has 8nm of radius. Moreover, in our case we have interaction between protein and a metal in water, and we can say that the protein doesn't generate plasmons. 

Try to find a paper where they also use this small distances.
The paper we used for comment 4 in reviewer 1 is a coarse-grained MD with NP near a membrane under an electric field and the distances are in the same range. 

https://pubs.rsc.org/en/content/articlepdf/2016/nr/c6nr02051h

"The electrostatic interactions were calculated using the particle mesh Ewald (PME) method with a real space cut off length of 1.2 nm and a fast Fourier-transform grid spacing of 0.24 nm"
Fig 5 shows distances between nanoparticle and membrane and they have values smaller than 1nm, even close to 0nm.

#### Reply
According to the reference cited, the model presented by the authors has a validity domain. For the case of metallic Ag and Au nanoparticles, the authors say: "We point out that our diffusive model is valid for structural dimensions exceeding the mean-free path that in pure single crystals can be of the order of 100 nm for Ag and Au... " In our case the nanoparticles are of 8nm of radius and the interactions are between a metallic nanoparticle and a protein that doesn't generate plasmons. 


#### Modifications
- no modifications needed

**Comment by cdcooper84** [comment]
### Comment 2
> 2. The superindex in the definition of V and K in Eqs. (8-9) should be corrected.
#### Ideas
- Yes, there is a inconsistency of superindices between Eq 8-9 with respect the rest of the text
- Replace the superindex by just \Gamma

#### Reply
The inconsistency in the superindices from Eq 8-9 was fixed by replacing `\mathbf{r}_\Gamma` for `\Gamma` .
 
#### Modifications
Fixed in commit 70757e3

**Comment by cdcooper84** [comment]
### Comment 3
>  3. The bold fonts for pi between eq.(22) and eq.(24) should be changed into plain.
#### Ideas
- The reviewer is right, but it's Eq 22b, 23 and 24
- In eq 23 and 24 there is a mixture of index and vector notation, we should choose one.
#### Reply
The bold fonts for p_i were removed. 
#### Modifications
Fixed in commit 9b620d7

**Comment by cdcooper84** [comment]
### Comment 4
> 4. Should the “location” in ”To study the effect of location of the analytes, ... Figure 13” (on page 9) be orientation?

#### Ideas
- Here we mean location, not orientation. We should explain how we get each configuration to the reviewer.

#### Reply
We are referring to the location not the orientation. We refer to the position on the x, y or z axis. The x and y configurations were obtain by performing a solid rotation of the z-configuration of 90 degrees along the x- and y-axis respectively. This statement was added in the text to clarify the point. 

#### Modifications
Add note on how we obtained these configurations 8180cb0

**Comment by cdcooper84** [comment]
### Comment 5
> 5. Figures (1) and (2) are basically the same which make the manuscript redundant. The same happens to Figs (3) and (7), and Figs (9) and (13)

#### Ideas
- I agree that Figs 1 and 2 are similar. But Fig 1 is better for an idea of the physics, Fig 2 is better to understand the model.
- Fig 3 and 7 are very similar. The only difference is that distances are defined, but we can do that in 3 too. (eliminate 7 and add d in 3)
- We don't agree with this, because the are placed in different position with respect to the incoming field. Perhaps we can add the direction of the electric field in the background to make that clearer. (add arrow pointing in the direction of electric field)

#### Reply
- Respect to Figure 1 and 2 we believe that Figure one represent more the physics while Figure 2 aims to explaining the model. Therefore we will opt to leave them. 
- Regarding Figure 3 and 7, we completely agree with the reviewer. We removed Figure 7 and adapted Figure 3. 
- Figure 9 and 13 do not represent the same the same disposition of the proteins, to make this point clear we adapted the figures by adding the direction of the electric field. 

#### Modifications
- We adapt Fig 3 and remove Fig 7. Changes made in e525375
- We add Electric field direction on display figures (old 9 and 13). Changes made in 8de6810

**Comment by cdcooper84** [comment]
### Comment 6
>  6. Is it possible to give more physical description of the bovine serum albumin proteins such as the distribution of the charge as implied in Eq.(12)? This I think may be useful to understand the effect of orientation of the molecule.
#### Ideas
- Related to comment 0 and 3 of reviewer 1. 
- We should add an explanation on the pipeline between the PDB and the mesh: explain pqr, SES, etc. 

#### Reply
This is related to Comment 0 of the first reviewer. In the original manuscript, it wasn't clear how the biomolecule is modeled with boundary integrals. This has been done previously with our code (citation [24]), however, we've added a section under Methods called 'Protein mesh preparation' explaining how the biomolecule is prepared, starting from its molecular structure in the Protein Data Bank, to the parameterization and physical meaning of the molecular surface definition. We also found a mistake in  the charge representation of Equation (12), which may have led to the confusion of the Reviewer, and the charge q_k was not being introduced accordingly.

#### Modifications
- Added 'Computational Workflow' section in commit fb1785c
- Added introduction of q_k variable in commit f648cd1d 
- Fixed Equation (12) in commit 4aa36fb

**Comment by cdcooper84** [comment]
### Comment 7
> 7. On page 11 the authors wrote ”Figure 10 shows a red shift of the plasmon resonance frequency peak in presence of the BSA proteins. This result agrees with experimental observations [42,43] ...”. In fact the red shift is easy to understand from a two-mode model and assume the mode provided by the BSA proteins is a lower energy mode with respect to the pamonic mode. So it is hard to say the experiment support the numerical calculation here. Maybe it is better to add in new calculation with well established software/technique to support the numerics here; this is just a very personal suggestion, and I would leave it to the author to decide.

#### Ideas
- The reviewer has a good point here. He/she wants better evidence that the code is correct.
We are not going to run other cases, but we should try to find better evidence of Silver NP and BSA, so far no luck. 

See if we can argument something more using these reference.
- This paper show concentrations of BSA, but I can't calculate how many are, since it is not clear in what volume are we sensing. They abotianed that the peak in resonance is something between 380-400nm  https://link.springer.com/content/pdf/10.1007%2Fs12274-017-1819-5.pdf

- This paper, show shifts on silver nanochips (no dimensions of nanochip provided) due to BSA (they say the concentration is 100 pM, but again I can't compute how many particles since I don't know in what volume are we sensing). The shift are smaller between 0.4 and 1 nm 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3937187/pdf/ijn-9-1097.pdf

 

#### Reply
- The reviewer claims the red shift we observe can be explained from the point of view of a two-mode model. We are not familiar with this theory, but if this relates to the references cited by the reviewer at the beginning of the report, it appears to be in a quantum-mechanics setting, which is outside the scope of our paper. The issue at hand, though, is how much the experimental results support our computational results. We remove the claim that the experimental results "agree" and simply state what they report and leave the reader to decide if they find the correspondence convincing enough.
- We delete the language "This result agrees with experimental observations" and state only what the experimental papers report. 
- Regarding adding other calculations with well-established software: 1) We provide evidence that our software solves correctly the mathematical model via grid convergence tests. 2) We are unable to run simulations with another open-source software because they cannot handle the amount of boundary elements we are using. The options of running with commercial software is not something we can do because we don't have access to such paid software.

#### Modifications

- Changes in 1556a72, df3c348 and b8a6e28
